src/rdo.rs

```diff
@@ -267,7 +267,9 @@ pub fn rdo_mode_decision(
   let mut mv_stack = Vec::new();
   let mode_context = cw.find_mvrefs(bo, LAST_FRAME, &mut mv_stack, bsize, false);
 
-  for &luma_mode in &mode_set {
+//  for &luma_mode in &mode_set {
+//  mode_set.par_iter().for_each(|&luma_mode| {
+  mode_set.iter().for_each(|&luma_mode| {
```

Comment:
I think I see similar change in your other recent PR, which you explained as a 'intra' or 'prep'?

Reply:
Yes, I'm trying to untangle bits so it is clearer, in the other set I used a proper struct instead of a tuple.